src/BoundaryConditions/flux_calculations.jl

```diff
@@ -0,0 +1,63 @@
+# General
+"""
+    get_boundary_flux(model, ::NoFluxCondition, _...)
+
+Construct zero boundary fluxes for a scalar or 2-element vector(x,y)
+"""
+function get_boundary_flux(model, ::NoFluxCondition, var::Fields.Field, Ym, Ya)
```

Comment:
I'll echo @jakebolewski on naming conventions: 

> One note, the convention used in Julia is to neglect get_ and have mutating set_ with a bang !:
> ex: `get_interior_values` -> interior_values / set_interior_values! 

(https://github.com/CliMA/LandHydrology.jl/pull/41#issuecomment-924184894)

Reply:
Good point. It depends on whether you're prescribing the flux or calculating it, so technically you could be either getting or setting!, depending on your boundary conditions. I was following the naming convention from the issue linked to this PR. I will note this in the TODO list above though (which I will later convert to an issue), and let the ClimaAtmos team decide.

Reply:
It seems the intent of this function here is clear, given some prognostic and auxiliary variables Y, Ya, Ym, "get" the appropriate boundary flux (`set` here would imply that the `ClimaCore.Operator` is aware of the b.c. at this point which isn't the case, right? ); the "set" operation is carried out with the corresponding `ClimaCore.Operators(SetValue...` or `ClimaCore.Operators(SetGradient...` e.g. line 66 in the single column example,  and then assigned to the appropriate `face` identifier. If you flag this as an issue we can make sure this is noted as we think about how ClimaSimulations is built.